docs/howto/remote-debugging.rst

```diff
@@ -0,0 +1,106 @@
+======================================
+How to debug an app in iOS and Android
```

Comment:
Agreed that this should be flagged as a "Pycharm" tutorial - especially given that one of the longer-term goals (and the original reason BeeWare started) is to develop a debugger :-)